fe/fe-core/src/main/java/com/starrocks/authentication/UserProperty.java

```diff
@@ -0,0 +1,18 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+package com.starrocks.authentication;
+
+import com.google.gson.annotations.SerializedName;
+
+public class UserProperty {
+    @SerializedName(value = "m")
```

Comment:
Set serializeName to "m" to save json size?

Reply:
Sí